src/Refactoring-DataForTesting/RBTransformationRuleTestData.class.st

```diff
@@ -280,6 +280,18 @@ RBTransformationRuleTestData >> checkMethod: aSmalllintContext [
 						classified: aSmalllintContext protocols]]
 ]
 
+{ #category : #accessing }
+RBTransformationRuleTestData >> class [
+
+	^ class
+]
```

Comment:
Usually it is a bad idea to have a method called "class" because it overrides the method allowing to get the class of an object.

For example, on compiled method. instead of having a method called "class" it was named "methodClass" so that we can still ask the class of the method.

Reply:
Yes, I'm aware. I only did it because this is a test fixture method. This will be removed in the next commit.